network-api/networkapi/wagtailpages/donation_modal.py

```diff
@@ -22,7 +23,28 @@ class DonationModal(TranslatableMixin, models.Model):
         default="Thanks for signing! While you're here, we need your help.",
     )
 
-    body = models.TextField(
+    body = RichTextField(
+        features=[
```

Comment:
Stakeholders would like to have all available featured enabled. See [comment](https://mozilla-hub.atlassian.net/browse/TP1-195?focusedCommentId=868400).